fetch.bs

```diff
@@ -1390,7 +1390,8 @@ to not have to set <a for=/>request</a>'s <a for=request>referrer</a>.
   <a>network error</a> if the request is not made to a same-origin URL.
 
   <dt>"<code>cors</code>"
-  <dd>Makes the request a <a>CORS request</a>. Fetch will return a <a>network error</a> if the
+  <dd>For requests whose <a for=request>response tainting</a> gets set to "<code>cors</code>", makes
+  the request a <a>CORS request</a> — in which case, fetch will return a <a>network error</a> if the
   requested resource does not understand the <a>CORS protocol</a>.
```

Comment:
Sorry, noticed this afterwards, but maybe you want to improve this as well since the resource might well understand it, but not wish to share.

Reply:
Went with: “or if the requested resource is one that intentionally does not participate in the CORS protocol”